android/PhysicalWeb/app/src/main/res/menu/main.xml

```diff
@@ -13,4 +13,9 @@
           android:orderInCategory="100"
           android:id="@+id/action_about"/>
 
+    <item android:title="@string/action_settings"
+          app:showAsAction="never"
+          android:orderInCategory="100"
```

Comment:
Could you give it a non-duplicate order?  it should probably be less than the about (and then it should also come before the about in this xml file)